test/sync/test_dataHandlers.js

```diff
@@ -142,7 +142,7 @@ module.exports = {
 
   'test set default collisionHandler': function(done) {
     var dataHandlers = dataHandlersModule({defaultHandlers: defaultHandlers});
-    dataHandlers.handleCollision(id, metaData, [],  function(err) {
+    dataHandlers.handleCollision(id, metaData, [], function(err) {
```

Comment:
@wei-lee I didn't think so as it looked to be calling a layer between this and the actual handler.
https://github.com/feedhenry/fh-mbaas-api/blob/FH-3123-sync-horizontal-scaling/lib/sync/dataHandlers.js#L236-L239
However, now I'm not sure.
I think it might be best to change the signature everywhere?